docs/reference_dataset/template_repository/hello-world-root-depls/pipeline-sample/concourse-pipeline-config/pipeline-sample.yml

```diff
@@ -1,20 +1,95 @@
 ---
 resources:
-- name: every-1m
-  type: time
-  source: {interval: 1m}
+- name: ((release-name))
```

Comment:
To get automatically loaded by COA, we need to add a var_file, automatically processed by bosh interpolate before loading pipeline into concourse.
sample-vars-tpl.yml (if spruce required) /  sample-vars.yml (without spruce):
```yaml
release-name:  my-release #
release-version: ((my-release-version)) #from crehub concourse-micro/main namespace 
git-username: (( grab secrets.git-username )) # from spruce secrets processing
....
```